src/raftstore/store/util.rs

```diff
@@ -764,6 +765,24 @@ impl Engines {
     }
 }
 
+pub fn format_split_info(region_id: u64, split_keys: &[Vec<u8>]) -> String {
```

Comment:
Why not introduce a struct that formats the comprehensive information about a `&[Vec<u8>]`? So you don't need to allocate a string for it. You may need to investigate `Display` trait.